.gitlab-ci.yml

```diff
@@ -303,8 +303,8 @@ deploy-ue1-tag:
   environment:
     name: parity-prod-ue1
 
-.validator-deploy: &validator-deploy
-  stage: publish
+.validator: &validator
+  stage: deploy
   dependencies:
     - build-linux-release
   image: parity/azure-ansible:v1
```

Comment:
what kind of image is it? where is the dockerfile and why is azure needed here?

Reply:
scripts/ansible/artifacts/polkadot  or substrate is taken from the artifacts of the build process

Reply:
This seems like the go to ansible image right now, but not needed. We just need a stable ansible container.

Reply:
> scripts/ansible/artifacts/polkadot or substrate is taken from the artifacts of the build process

okay, and what about the empty file in this pr?

> This seems like the go to ansible image right now, but not needed. We just need a stable ansible container.

how is it built or maintained? is there an advantage over using one of https://hub.docker.com/u/ansible/ ?

Reply:
Will remove the empty dir, this was local testing (having an artifact binary present).